### PR TITLE
Block number and hash in the gas estimation msg

### PR DESCRIPTION
**Pull request by hackaugusto**
This information is useful for debugging race conditions https://github.com/raiden-network/raiden/issues/5744

### CODE REVIEW

File: raiden/network/proxies/token_network_registry.py [hunk 1]
```diff
@@ -383,14 +383,17 @@ def _add_token(
                     "though the gas estimation succeeded."
                 )
 
-            token_network_address = self.get_token_network(token_address, receipt["blockHash"])
+            succeeded_at_blockhash = receipt["blockHash"]
```
Comment (palango) [inline raiden/network/proxies/token_network_registry.py]: Is this bytes or str?

Comment (hackaugusto) [reply raiden/network/proxies/token_network_registry.py]: I believe it is bytes, since in other places we are calling `to_hex` on this value.

Comment (palango) [reply raiden/network/proxies/token_network_registry.py]:
```
[50] > /Users/paul/Projects/brainbot/raiden/raiden/network/rpc/client.py(697)poll()
-> is_transaction_confirmed = block_number >= confirmation_block
   5 frames hidden (try 'help hidden_frames')
(Pdb++)
(Pdb++) tx_receipt
AttributeDict({'blockHash': HexBytes('0x07f464d8366f08a8a000556b11c4bcd573e5677b4d697b5406e8cfb0c965e049'), 'blockNumber': 12, 'contractAddress': '0x3ed9588530D2E3fFeEa0a37E090c33A1C93d17e6', 'cumulativeGasUsed': 298578, 'from': '0x504dabd4c3cb867286404c90b03cf1ed2d8d1607', 'gasUsed': 298578, 'logs': [], 'logsBloom': HexBytes('0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000'), 'status': 1, 'to': None, 'transactionHash': HexBytes('0xb0ca9373ab9c5e6c3b9c897c12451be238c0330a0d5344acfdc70b428bfb48fd'), 'transactionIndex': 0})
(Pdb++) tx_receipt['blockHash']
HexBytes('0x07f464d8366f08a8a000556b11c4bcd573e5677b4d697b5406e8cfb0c965e049')
```

not sure

Comment (hackaugusto) [reply raiden/network/proxies/token_network_registry.py]: `HexBytes` is web3 magic that prints a bytes object as hex. https://github.com/ethereum/hexbytes

**Pull request merged**